### ISSUE TITLE
Docker build failed due to github inaccessibility in China

### ISSUE THREAD

**Issue by shupx** [larics/icuas22_competition#15]
I failed to build the docker when running [docker_build.sh](https://github.com/larics/icuas22_competition/blob/main/docker_build.sh) because I can not access github in china internet directly. So I failed to download two ROS dependencies in [Dockerfile.bionic](https://github.com/larics/icuas22_competition/blob/main/Dockerfile.bionic) which are
 `https://github.com/larics/larics_gazebo_worlds.git`
`https://github.com/larics/storm_gazebo_ros_magnet.git `

I managed to solve the problem by cloning these two github repositories to my gitee accout which is accessible in china. They are:
`https://gitee.com/shu-peixuan/larics_gazebo_worlds.git`
`https://gitee.com/shu-peixuan/storm_gazebo_ros_magnet.git`

So if anyone had a same problem with me, **just replace the github url in the [Dockerfile.bionic](https://github.com/larics/icuas22_competition/blob/main/Dockerfile.bionic)  with the gitee url listed above.**
Thanks again to the organizer and developer of this competition. 
Good luck!

**Comment by fpetric** [comment]
@shupx thanks for the update and the fix. we will leave this open for others to see.

**Comment by caozx1110** [comment]
I solve the problem by changing the "https://" of website in dockerfile to "git://", lol
